shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/TCLStatement.g4

```diff
@@ -20,7 +20,10 @@ grammar TCLStatement;
 import Symbol, Keyword, OracleKeyword, Literals, BaseRule;
 
 setTransaction
-    : SET TRANSACTION
+    : SET TRANSACTION ((READ (ONLY | WRITE)
```

Comment:
The rule is not correct.
![image](https://user-images.githubusercontent.com/22066046/120063688-05850680-c09b-11eb-9ff7-0664bbe34f69.png)

Reply:
Hi @wgy8283335, could you please specify which point and why it's not correct?

Reply:
It's my miss understanding.